index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>The new york times</title>
+        <link href="css/index.css" rel="stylesheet" />
+        <link rel="shortcut icon" href="img/new-york-times-icon.png" />
+    </head>
+    <body>
+
+        <header>
+            <nav>
+                <ul class="nav">
+                    <li class="nav-link list"><a href=""><img src="img/list.svg" alt="menu" /></a></li>
+                    <li class="nav-link search"><a href=""><img src="img/search.svg" alt="search" /></a></li>
+                    <li class="nav-link title"><a href="" id="logo_link">Space & Cosmos</a></li>
+                    <li class="nav-link logo"><a href=""><img src="img/Logo.png" alt="logo"/></a></li>
+                    <li class="nav-link sub"><input id="sub_but" type="button" value="SUBSCRIBE NOW"/></li>
+                    <li class="nav-link log"><input id="log_but" type="button" value="LOG IN"/></li>
```

Comment:
[required] Make sure that all of `href` atttributes of `<a>` tags have a proper value, **not empty**, as a peace of advice instead of letting them like `href=""` you can put a `#`, =>  `href="#"`.